spec/controllers/internal_api/v1/pillars_controller_spec.rb

```diff
@@ -310,6 +310,9 @@
       {
         registries:          [],
         system_certificates: [],
+        dex:                 {
+          connectors: []
+        },
```

Comment:
I think this is unrelated with this PR, isn't it?

Reply:
It will disappear after a rebase. An old PR was merged without rebasing and introduced this small issue that was already fixed.

Reply:
lol

Reply:
Yes, this was in purpose because travis was red because of this. Rebasing now.